charts/openebs/templates/cleanup-webhook.yaml

```diff
@@ -23,10 +23,6 @@ spec:
         app: {{ template "openebs.name" . }}
     spec:
       serviceAccountName: {{ template "openebs.serviceAccountName" . }}
-      {{- if .Values.webhook.tolerations }}
```

Comment:
@shovanmaity - let us fix the indentation issue and keep the tolerations. In some setups, every node is tainted like in openshift and would need admin to provide a toleration for job or pod to run.

Reply:
Also, can this fix be taken into the 2.10 helm chart.

Reply:
@kmova  as this patch is added in PR #224 I am closing this PR.